sky/setup_files/setup.py

```diff
@@ -233,6 +234,10 @@ def parse_readme(readme: str) -> str:
     'oci': ['oci'] + local_ray,
     'kubernetes': ['kubernetes>=20.0.0'] + local_ray,
     'remote': remote,
+    'vsphere': [
+        'pyvmomi==8.0.1.0.2',
+        'vsphere-automation-sdk @ git+https://github.com/vmware/vsphere-automation-sdk-python.git@v8.0.1.0'
+    ] + local_ray,
```

Comment:
`local_ray` should not be required as we are not using ray autoscaler locally.
```suggestion
    ],
```

Reply:
Removed